scripts/ci/gitlab/pipeline/publish.yml

```diff
@@ -154,7 +154,7 @@ publish-rustdoc:
     - .kubernetes-env
   image:                           paritytech/tools:latest
   variables:
-    GIT_DEPTH:                     100
+    GIT_DEPTH:                     0
```

Comment:
Just curious, why do we need this in the `publish-rustdoc` job?

Reply:
Hm, probably we don't :D removed